### PR TITLE
Backend models

### PR DESCRIPTION
**Pull request by phemmylintry**

### CODE REVIEW

File: app/models/models.py [hunk 1]
```diff
@@ -17,7 +21,7 @@ class Village(db.Base):
     __tablename__ = "villages"
     id = Column(String(255), primary_key=True, index=True, default=uuid4().hex)
     name = Column(String(255), index=True)
-    loation = Column(String(255), ForeignKey("locations.id"))
+    location_id = Column(String(255), ForeignKey("locations.id"))
```
Comment (nseetim) [inline app/models/models.py]: You are certain that the tablename is called `locations`?, to enable code readability i would suggest using the format `ForeignKey(Location.id)` as opposed to `ForeignKey("locations.id")`

File: app/models/models.py [hunk 2]
```diff
@@ -26,7 +30,7 @@ class Voter(db.Base):
     id = Column(String(50), primary_key=True, index=True, default=uuid4().hex)
     village = Column(String(255), ForeignKey("villages.id"))
     name = Column(String(255), index=True)
-    contact = Column(String(255), ForeignKey("contacts.id"))
+    contact = Column(String(255), ForeignKey("contactus.id"))
```
Comment (nseetim) [inline app/models/models.py]: You are certain that the tablename is called `contactus`?, to enable code readability i would suggest using the format `ForeignKey(ContactUs.id)` as opposed to `ForeignKey("contactus.id")`

File: app/models/models.py [hunk 3]
```diff
@@ -35,7 +39,7 @@ class Voter(db.Base):
 
 class DuplicatedVoter(db.Base):
     __tablename__ = "duplicated_voters"
-    pass
+    id = Column(String(50), primary_key=True, index=True, default=uuid4().hex)
```
Comment (nseetim) [inline app/models/models.py]: Like earlier mentioned lets have a constant `UUID_HEX=uuid4().hex` defined at the top of the module that we can reuse, lets keep it DRY

File: app/models/models.py (hunk 1 above)
Comment (phemmylintry) [reply app/models/models.py]: yea, that's the table name, according to how it's been declared in the package. but sure this method make it more readable

**Review by nseetim — APPROVED** [review]
Nicely done

**Pull request merged**